src/shell/log.c

```diff
@@ -337,7 +337,7 @@ static int log_setlevel (flux_shell_t *shell, const char *dest, int level)
 
 int flux_shell_log_setlevel (int level, const char *dest)
 {
-    if (level < FLUX_SHELL_FATAL || level > FLUX_SHELL_TRACE) {
+    if (level < -1 || level > FLUX_SHELL_TRACE) {
```

Comment:
Might be more clear if something like `FLUX_SHELL_DISABLE = -1` were added to the enum.

Reply:
Good point, but can you use a negative value with an enum?

Reply:
It would appear so :-)

Reply:
Ok, I'll add that.